### ISSUE TITLE
Failed to execute phyton plugins: ipsec

### ISSUE THREAD

**Issue by Pr1mEzz** [bashclub/check-opnsense#12]
Hi,

thank you for this amazing Implementation.

I have the following Problems:
[Check_MK Discovery] [agent] Version: 0.88, OS: OPNsense, Failed to execute python plugins: ipsec, execution time 6.5 sec
![image](https://user-images.githubusercontent.com/108271267/175927648-f61d4aa8-e9e8-4875-9572-055ad54ecfc6.png)

This error is flapping (cannot describe exact time intervals) and leads to other errors in IPSec-Tunnel monitoring: 

[IPsec Tunnel] UNKNOWN - Item not found in monitoring data

After some minutes, the status is OK again.

Firmware of my device: 
Version 21.10 Config changed: 13:05 24.06.2022, update_available: 0.00, last_updated: 22164966.00, apply_finish_time: 258614.00

Thanks in advance for support!

**Comment by JaapHoetmer** [comment]
Hi,

Thanks from me as well for this great development.
We are experiencing the same issue, but with OpenVPN, and only on one firewall. The others don't experience this behaviour. So it seems it is not consistent.

I tried to download the latest version (0.96) on this device to see if that resolved the issue, but it doesn't.
I have cleared the logs, but that didn't help either.

Check_MK Agent | Version: 0.96, OS: OPNsense, Failed to execute python plugins: openvpn

Firmware of the OpnSense device: 22.7.4
Version of CheckMK: 2.1.0p1

Kind regards,
Jaap
